docs/prow/crier.md

```diff
@@ -0,0 +1,177 @@
+    job_types_to_report:
+      - presubmit
+      - postsubmit
+    # default: None
+    job_states_to_report:
+      - failure
+      - error
+    # required
+    channel: my-slack-channel
+    # The template shown below is the default
+    report_template: "Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <{{.Status.URL}}|View logs>"
+
+  # "org/repo" slack config
+  istio/proxy:
+    job_types_to_report:
+      - presubmit
+    job_states_to_report:
+      - error
+    channel: istio-proxy-channel
+
+  # "org" slack config
+  istio:
+    job_types_to_report:
+      - periodic
+    job_states_to_report:
+      - failure
+    channel: istio-channel
+```
+
+The Slack `channel` can be overridden at the ProwJob level via the `reporter_config.slack.channel` field:
+```yaml
+postsubmits:
+  some-org/some-repo:
+    - name: example-job
+      decorate: true
+      reporter_config:
+        slack:
+          channel: 'override-channel-name'
+      spec:
+        containers:
+          - image: alpine
+            command:
+              - echo
+```
+
+## Current Slack notification settings
+
+Crier does not send Slack notifications at all for presubmit jobs.
+
+Reporter config:
+```
+job_types_to_report:
+  - postsubmit
+  - periodic
+  - batch
+```
+
+If you don't want to report postsubmit or periodic jobs to report to Slack channel use `skip_report:true`.
+If the job is still in testing phase we can set `optional: true`.
+
+## Migration from plank for github report
+
+First, you need to disable GitHub reporting in Plank, add the `--skip-report=true` flag to the Plank [deployment](https://github.com/kyma-project/test-infra/blob/master/prow/cluster/components/11-plank_deployment.yaml).
+
+Before migrating, be sure plank is setting the [PrevReportStates field](https://github.com/kubernetes/test-infra/blob/de3775a7480fe0a724baacf24a87cbf058cd9fd5/prow/apis/prowjobs/v1/types.go#L566)
+by describing a finished presubmit ProwJob. Plank started to set this field after commit [2118178](https://github.com/kubernetes/test-infra/pull/10975/commits/211817826fc3c4f3315a02e46f3d6aa35573d22f), if not, you want to upgrade your plank to a version includes this commit before moving forward.
+
+Flags required by Crier:
+- Point `config-path` and `--job-config-path` to your prow config and job configs accordingly.
+- Set `--github-worker` to be number of parallel github reporting threads you need.
+- Point `--github-endpoint` to ghproxy, if you have set that for plank.
+- Bind github oauth token as a secret and set `--github-token-path` if you've have that set for plank.
+
+In your plank deployment, you can:
+- Remove the `--github-endpoint` flag.
+- Remove the github oauth secret, and `--github-token-path` flag if set.
+- Add`--skip-report`, so plank will skip the reporting logic.
+
+Both change should be deployed at the same time, if have an order preference, deploy crier first since report twice should just be a no-op.
```

Comment:
```suggestion
Both changes should be deployed at the same time. If you need to deploy them sequentially, deploy Crier first to avoid double-reporting.
```
This sentence is very unclear. I'm not sure if my suggestion reflects its original meaning. If not, let's discuss it.